### ISSUE TITLE
Relic of the Pact, The Squire and Essence Worm crash PoB

### ISSUE THREAD

**Issue by 5Karma** [PathOfBuildingCommunity/PathOfBuilding#3796]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### What is the expected behaviour/value?

Relic of the Pact, The Squire and Essence Worm don't crash PoB when used together

### What is the actual behaviour/value?

When using Relic of the Pact, The Squire and Essence Worm PoB often crashes with a lua error

### How to reproduce the issue

1.) Open new build
2.) Add and equip Relic of the Pact
3.) Add and equip The Squire
4.) Add and equip Essence Worm
5.) Click the Dropdown Menu for the ring slot where Essence Worm is equipped
6.) Close the Dropdown Menu by clicking outside
(This Step 5/6 is not the only way to trigger the error message but its the most consistent one i found after a couple minutes of testing. Clicking around between the different panels in the items tab also seems to work but is not as consistent.)

### Build code

```shell
https://pastebin.com/Rb3DwUyL
```

### Screenshots

https://i.imgur.com/zxBIBKD.png

**Comment by Sadakandras** [comment]
I came to post this same bug as well.
"The Squire" is not needed to replicate this bug.
The order in which you add "Relic of the Pact" and "Essence Worm" does not matter.
I never clicked any dropdown menus. For me a couple of seconds after adding either "Relic of the Pact" or "Essence Worm", the error message pops up.

**Build Code**

https://pastebin.com/mFncwpPn

**My screenshot if it helps.**

https://i.imgur.com/O1OcWBH.png

**Comment by Sadakandras** [comment]
Ok it seems "Essence Worm" is not necessarily required to replicate the bug, I copied the "80% reduced Reservation Efficiency" from "Essence Worm" over to make a custom rare ring with the full "80% reduced Reservation Efficiency" and had the same issue.
So I tried using the "Memory Vault Praetor Crown" for it's 20% reduced Reservation Efficiency and had no problems.
So then I tried changing the numbers on the custom items I made to find the limit, I was able to safely get to "59% reduced Reservation Efficiency" without the bug report pop-up.
I also tried adding smaller values on separate items and I still got the bug upon reaching 60% or more reduced Reservation Efficiency.
So then I tried allocating an aura node mastery with "30% increased Life Reservation Efficiency of Skills" and was then able to add more reduced Reservation Efficiency to the items I was wearing until reaching a complete total of 60%(90% from gear) again.

It seems Relic of the pact just breaks PoB at a global total of 60% reduced Reservation Efficiency
